pyproject.toml

```diff
@@ -28,7 +28,7 @@ salib = "^1.5.1"
 torcheval = "^0.0.7"
 anytree = "^2.12.1"
 torchmetrics = "^1.7.1"
-
+pyro-ppl = "^1.9.1"
```

Comment:
I tried running the notebook and run into issues because `corner` is not installed.

Reply:
Added in [4c87eda](https://github.com/alan-turing-institute/autoemulate/pull/505/commits/4c87eda789bb92df200cbf2ead7e2950b8297cd2)